package.json

```diff
@@ -79,8 +79,8 @@
     "react-native-image-pan-zoom": "^2.1.12",
     "react-native-image-picker": "^2.3.3",
     "react-native-keyboard-spacer": "^0.4.1",
-    "react-native-onyx": "git+https://github.com/Expensify/react-native-onyx.git#d0a5d12c08f2f029e3038fb3516a93a9403e9746",
     "react-native-modal": "^11.10.0",
+    "react-native-onyx": "git+https://github.com/Expensify/react-native-onyx.git#d0a5d12c08f2f029e3038fb3516a93a9403e9746",
```

Comment:
The version here wasn't changed (the commit hashes are the same), `npm install` simply changed the location of this when I ran it (I guess it sorted everything alphabetically).